spec/rubyspecs

```diff
@@ -112,6 +112,7 @@ rubyspec/language
 !rubyspec/language/next_spec
 !rubyspec/language/predefined_spec
 !rubyspec/language/proc_spec
+!rubyspec/language/rescue_spec
```

Comment:
Unfortunately we lost the ability to run language/rescue spec due to https://github.com/opal/opal/issues/1305 :cry: